### ISSUE TITLE
Move configurable data to config file

### ISSUE THREAD

**Issue by iamtomhewitt** [iamtomhewitt/home-dashboard#6]
For example, weather latitude and longitude can go in a separate config section:

```json
...,
"weather": {
    "latitude": 123,
    "longitude": 123
}
```

#### Other Config
* Bin start dates
* Rename the Todoist variable to `projectId` instead of `todoistProjectKeyName`
* Train station code

**Comment by iamtomhewitt** [comment]
#### Other Config
* Rename the Todoist variable to `projectId` instead of `todoistProjectKeyName`
* Train station code

**Comment by iamtomhewitt** [comment]
Fixed in 3.1.2
